src/libs/ajax.js

```diff
@@ -410,12 +410,12 @@ const Buckets = signal => ({
     return _.filter(({ name }) => name.endsWith('.ipynb'), items)
   },
 
-  notebook: (namespace, bucket, name) => {
+  notebook: (namespace, bucket, newBucket, name) => {
```

Comment:
Since this is specific to `copy`, it should probably be a parameter there instead of at this level.